tooling/template/app/[[...permalink]]/page.tsx

```diff
@@ -28,15 +28,17 @@ const lastUpdated =
 const getSchema = async (
   permalink: DynamicPageProps["params"]["permalink"],
 ) => {
+  // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
   if (permalink && permalink.length > 0 && typeof permalink !== "string") {
     const joinedPermalink = permalink.join("/")
 
     const schema = (await import(`@/schema/${joinedPermalink}.json`).then(
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-return, @typescript-eslint/no-unsafe-member-access
       (module) => module.default,
     )) as IsomerPageSchema
 
     const lastModified =
-      // @ts-ignore blah
+      // @ts-expect-error to fix when types are proper
```

Comment:
can add a `// TODO:` so we can search by this sentinel string?

Reply:
i do in last PR